cla_backend/libs/eligibility_calculator/cfe_civil/proceeding_types.py

```diff
@@ -0,0 +1,23 @@
+CATEGORY_TO_PROCEEDING_TYPE = {
+    "immigration": {
+        "ccms_code": "IM030",
```

Comment:
I think this is the wrong proceeding type - I think we need IA031 for asylum cases

Reply:
@starswan We need to use **[IM030](https://github.com/ministryofjustice/cfe-civil/blob/e86f68ba8d5a3d04bc5ff0f122437696b95ac71a/app/constants/cfe_constants.rb#L14)** for category "immigration"